app/models/profile.rb

```diff
@@ -0,0 +1,59 @@
+class Profile < ActiveRecord::Base
+  include Privacy
+
+  belongs_to :user
+
+  has_attached_file :curve, styles: {
```

Comment:
> Do we know yet what the contents of these profiles will look like?

They will be profiles containing numbers in EUR which don't need to be scaled indeed :smile: